cs/dnc/src/json/project.json

```diff
@@ -37,20 +37,28 @@
     "dependencies": {
         "attributes": "1.0.0-*",
         "core": "1.0.0-*",
-        "Newtonsoft.Json": {
-            "version": "9.0.1"
-        },
         "reflection": "1.0.0-*"
     },
     "frameworks": {
         "netstandard1.0": {
             "dependencies": {
-                "NETStandard.Library": "1.6.0"
+                "NETStandard.Library": "1.6.0",
+                "Newtonsoft.Json": {
+                    "version": "9.0.1"
+                }
             }
         },
         "netstandard1.6": {
+            "buildOptions": {
+                "define": [
+                    "SUPPORTS_BIGINTEGER"
+                ]
+            },
             "dependencies": {
-                "NETStandard.Library": "1.6.0"
+                "NETStandard.Library": "1.6.0",
+                "Newtonsoft.Json": {
+                    "version": "10.0.1"
```

Comment:
Version 10.0.1 picked up support for BigInteger for .NET Standard 1.1 or greater.